plugin-networkmonitor/lxqtnetworkmonitorconfiguration.cpp

```diff
@@ -86,7 +88,7 @@ void LXQtNetworkMonitorConfiguration::loadSettings()
         ui->interfaceCB->addItem(QLatin1String(stats[ix].interface_name));
 
     QString interface = settings().value(QStringLiteral("interface")).toString();
-    ui->interfaceCB->setCurrentIndex(qMax(qMin(0, count - 1), ui->interfaceCB->findText(interface)));
+    ui->interfaceCB->setCurrentIndex(std::clamp(count - 1, ui->interfaceCB->findText(interface), 0));
```

Comment:
`QComboBox::findText` can be > 0, in which case, `clamp` will be undefined. So, `min` and `max` should be used.